<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjetoFinal</class>
 <widget class="QMainWindow" name="ProjetoFinal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjetoFinal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="logText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>521</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>video original</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>video processado</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>341</y>
      <width>81</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="FuturePastLayout">
     <item>
      <widget class="QCheckBox" name="futureButtom">
       <property name="text">
        <string>Future</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="pastButtom">
       <property name="text">
        <string>Past</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="speedLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="speedSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="closeButtom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encerrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runDemoKalman">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RunDemoKalman</string>
    </property>
   </widget>
   <widget class="QComboBox" name="videoFileOption">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>178</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="PauseRestartLayout">
     <item>
      <widget class="QPushButton" name="pauseButtom">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restartButtom">
       <property name="text">
        <string>Restart</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clearButtom">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>360</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>611</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="inputImg">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="outputImg">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>outputImg</zorder>
   <zorder>inputImg</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>logText</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>closeButtom</zorder>
   <zorder>runDemoKalman</zorder>
   <zorder>videoFileOption</zorder>
   <zorder>clearButtom</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
